src/components/Hr/Hr.tsx

```diff
@@ -12,7 +12,7 @@ export interface Props {
    * Size variations:
    * - **lg** yields a thicker horizontal rule
    */
-  size?: 'lg' | 'brand';
```

Comment:
This was fixed in https://github.com/chanzuckerberg/edu-design-system/pull/967 but that PR was long forgotten and buried beneath centuries of commits, so I figured it's just easier to fix it in a new PR.